lib/ecto/query/api.ex

```diff
@@ -272,6 +272,49 @@ defmodule Ecto.Query.API do
   def fragment(fragments), do: doc! [fragments]
 
   @doc """
```

Comment:
Please use `~S"""` this way you don't need to escape the interpolations down below. :)